src/metabase/driver.clj

```diff
@@ -567,7 +582,10 @@
     :index-info
 
     ;; Does the driver support a faster `sync-fks` step by fetching all FK metadata in a single collection?
-    :describe-fks})
+    :describe-fks
+
+    ;; Does the driver support a faster `sync-fields` step by fetching all FK metadata in a single collection?
```

Comment:
nice idea, since the feature name is closely coupled to the multimethod I can't see this comment becoming out of date. done 0b1742f0c8289bd97850a477f5f6aa171ac1ae30